Resources/Prototypes/Entities/Objects/Tools/matches.yml

```diff
@@ -72,6 +72,14 @@
   - type: Item
     sprite: Objects/Tools/matches.rsi
     heldPrefix: matchbox
+    size: 5
+  - type: Storage
+    capacity: 6
+    whitelist:
```

Comment:
literally 1984 we cant put nuke disk (somewhat reasonable) or pens (very reasonable) in matchbox